src/utils/statens_grunnbeloep.ts

```diff
@@ -0,0 +1,3 @@
+const statens_grunnbeloep: number = 101351; // g-beløp per 29. januar 2021
```

Comment:
Dette gjelder ikke lenger. Gjaldt frem til 1. mai, hvis de har oppdatert enda.... Kanskje ikke noe for et fellesbibliotek?